### ISSUE TITLE
Same Username For Two Accounts

### ISSUE THREAD

**Issue by mattbaseball** [GameFaceChat/GameFace#16]
**Describe the bug**
It appears there is no check for if the username is in the system or not. When making another account with my email, I tried to have the same username. This logged me out of my first account and doesn't seem to allow me back in. 

**To Reproduce**
1. Sign up with email
2. Create account and give it a username which is already taken
3. Try logging into your previous account with that username

**Expected behavior**
Error message stating that the username is already in use.

**Smartphone (please complete the following information):**
 - Device: Pixel 3a Emulator and Samsung Galaxy S9
 - API Level: API 30

**Comment by sriharivishnu** [comment]
Should be fixed in v0.0.7

**Comment by mattbaseball** [comment]
Yes this is indeed fixed in v0.0.7. Just also needs to be able to identify it as the same username when the capital letters are in different places, i.e. Hello is recognized as the same user as hEllo. Feel free to close this issue once that is resolved.
